src/frontend/src/config.ts

```diff
@@ -0,0 +1,7 @@
+/* Global configuration for II and constants */
+
+// The URL where the official, production II is served
+export const OFFICIAL_II_URL = "https://identity.ic0.app";
```

Comment:
Shouldn't this include `OFFICIAL_II_URL_NO_PROTOCOL`?

Reply:
Yeah first did that but it seems like a lot of noise for not much benefit

Reply:
actually no, I did the oppose: stripped `https://` from `OFFICIAL_II_URL` to make `OFFICIAL_II_URL_NO_PROTOCOL`. What you suggest is actually much nicer